vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold.cc

```diff
@@ -71,17 +85,13 @@ void Uphold::ContributionCompleted(
   const auto amount = ConvertToProbi(std::to_string(reconcile.fee_));
 
   if (result == ledger::Result::LEDGER_OK) {
-    // 5% fee
-    auto fee_callback = std::bind(&Uphold::OnFeeCompleted,
-                              this,
-                              _1,
-                              _2,
-                              viewing_id);
-
-    transfer_->Start(fee,
-                     GetFeeAddress(),
-                     ledger::ExternalWallet::New(wallet),
-                     fee_callback);
+    const auto current_time_seconds = base::Time::Now().ToDoubleT();
+    auto transfer_fee = ledger::TransferFee::New();
+    transfer_fee->id = viewing_id;
+    transfer_fee->amount = fee;
+    transfer_fee->execution_timestamp =
+        current_time_seconds + brave_base::random::Geometric(60);
```

Comment:
Is 60s arbitrary, or a pre-determined/required value?

Reply:
required value